backtester/algorithms/Simple.cpp

```diff
@@ -1,14 +1,17 @@
 #include "Simple.h"
+#include <string>
 
-Simple::Simple(const unordered_map<string, double> params)
-{
+Simple::Simple() {
+	bought = false;
```

Comment:
Maybe we should initialize the parameters in a different way. It is possible to set them using this format:
Simle::Simple() 
         : bought(false)
{

}

That way you separate the logic from the variable initialization + they'll be easy to find